mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1666,6 +1675,11 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		ResourceLabels: expandStringMap(d, "resource_labels"),
 <% unless version == 'ga' -%>
 		NodePoolAutoConfig: expandNodePoolAutoConfig(d.Get("node_pool_auto_config")),
+<% end -%>
+<% if version == 'beta' -%>
```

Comment:
`<% unless version == 'ga' -%>` is the same as `<% if version == 'beta' -%>`, just weird ruby-isms. We can save a couple of lines throughout the file by merging the two statements

Reply:
Changed it to `<% unless version == 'ga' -%>` everywhere in this file.